pkgs/by-name/sk/skimpdf/package.nix

```diff
@@ -0,0 +1,37 @@
+{ stdenv
+, lib
+, undmg
+, fetchurl
+,
+}:
+stdenv.mkDerivation rec {
+  pname = "Skim";
+  version = "1.6.21";
```

Comment:
new version released https://sourceforge.net/projects/skim-app/files/Skim/

Reply:
The new version will be added in a new pr, like nixpkgs wants it to be